apps/samplePoint/plot2D.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python
+"""
+Usage: plot2D.py --input data.txt [--output graph.png]
+
+Tool to plot as a 2D graph the evolution of the data at a specific point over
+time
+"""
+
+import argparse
+import numpy as np
+import matplotlib.pyplot as plt
+
+__author__ = "Jafet Villafranca"
+__email__ = "jafet.villafrancadiaz@epfl.ch"
+__copyright__ = "Copyright 2016, EPFL/Blue Brain Project"
+
+
+def main():
+
+    parser = argparse.ArgumentParser(description="Tool to plot as a 2D graph "
+                                     "the evolution of data over time, loading "
+                                     "the values from file")
+    parser.add_argument("-i", "--input", help="input file containing "
+                        "all the values to plot: one line per value, using "
+                        "the 'timestamp value' format")
+    parser.add_argument("-o", "--output", help="output file name to save the "
+                        "resulting 2D graph as an image. If not specified, "
+                        "open a window showing an interactive plot")
+    args = parser.parse_args()
+
+    if not args.input:
+        parser.print_help()
+        exit()
+
+    x = []
+    y = []
+    for line in open(args.input):
+        l = line.strip()
+        if l and not l.startswith("#"):
+            values = line.split()
+            if len(values) != 2:
+                continue
+            x.append(float(values[0]))
+            y.append(float(values[1]))
```

Comment:
If the conversion fails this will raise an exception. Print a more user friendly message than the exception?